html/interaction/focus/the-autofocus-attribute/document-with-fragment-valid.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="resources/utils.js"></script>
+
+<iframe src="resources/frame-with-autofocus-element.html#anchor1"></iframe>
+
+<script>
+'use strict';
+
+promise_test(async () => {
```

Comment:
The fact that these tests have two pass conditions is a bit sad, but I can't think of a way to make it better.

However, it might be worth adding a test where instead of using a frame with an autofocus element, you used a frame without one (but with an anchor). Then you insert the autofocus element into the frame later, after the navigation and scrolling-to-fragment completes. If I understand correctly, that will definitely fail to autofocus.

Reply:
That's a good idea.  Done.